fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetaClient.java

```diff
@@ -293,28 +311,31 @@ private List<Partition> getPartitionsWithRetry(String dbName, String tableName,
         } catch (Exception e) {
             throw new StarRocksConnectorException("Failed to getPartitionsNames on [%s.%s], msg: %s",
                     dbName, tableName, e.getMessage());
+        } finally {
+            if (client != null) {
+                client.close();
```

Comment:
client.finish()?

Reply:
yes, the getPartitionsWithRetry is easy to cause some error. so we remove it every time.

Reply:
I don't get this one. What error?